ssr/render.js

```diff
@@ -145,6 +145,13 @@ export default function render(
   const webfontURLs = extractWebFontURLs();
   const $ = cheerio.load(buildHtml);
 
+  // Some day, we'll have the chome localized and then this can no longer be
+  // hardcoded to 'en'. But for now, the chrome is always in "English (US)"
+  // and the 'lang' attribute should always be expressed in  ISO 639. (e.g 'en'
+  // or 'zh' (not 'en-US' or 'zh-CD))
+  // https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
+  $("html").attr("lang", "en");
```

Comment:
This is a change from prior MDN convention. Prior to the launch of Yari, we always used the full locale, so `en-US` instead of `en` (see below for my local version of the Kuma-generated home page):

<img width="574" alt="image" src="https://user-images.githubusercontent.com/3743693/108435401-346b2100-71fe-11eb-88a5-ebb69c1bc888.png">

It seems to me that we should use the full locale here as well, since any future chrome translations could vary based on the country code, for example `zh-CN` may have different chrome translations than `zh-TW`.

Reply:
That's not what I see:
<img width="789" alt="Screen Shot 2021-02-19 at 9 12 05 AM" src="https://user-images.githubusercontent.com/26739/108515008-970dfc80-7292-11eb-9ddd-b1d0ad0ec5fb.png">

<img width="820" alt="Screen Shot 2021-02-19 at 9 12 59 AM" src="https://user-images.githubusercontent.com/26739/108515067-ad1bbd00-7292-11eb-995a-2726c8b182eb.png">

It's clear that the old Kuma home page (replaced as of a couple of days ago) does it differently from a document (react) page. In fact, in `react_base.html` you can see it uses the `LANG` context variable. (from the `i18n` context processor) which is, for example, `zh-CN`. 
But in `react_document.html` it uses: `lang="{{ document_data['hrefLang'] }}" `

So, my initial attempt on this PR did it the way Kuma (documents) did it. 
But the more I read the documentation ([for example](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)) I see that they use the full format. I.e. language and region. 

It's hard to find reliable prior art, but https://developers.google.com/search/docs/advanced/crawling/localized-versions (switch language to Portuguese Brasilian) and you'll find `lang="pt-BR"`.

It seems to me that it's quite flexible. For example, on [our own documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) it brings up an interesting point about the [`:lang`](https://developer.mozilla.org/en-US/docs/Web/CSS/:lang) pseudo-class. If someone uses uses `:lang(zh)` it *will* match if the HTML/DOM was `lang="zh-CN"`. So it's quite flexiable.

The more I read the documentation it feels like this:
- You *can* use `en` or `en-US`. 
- If you have a region (e.g. `US` or `BR`), feel free to use it

Reply:
In conclusion, let's stick with `lang="zh-CN"` which *is* a change from how Kuma was doing it (apart from the home page).